index.js

```diff
@@ -29,7 +30,7 @@ var isLiteral = require('nlcst-is-literal');
  *   break the middleware.
  */
 function all(tree, file, config, next) {
-    var hun = config.dictionary;
+    spellchecker.use(config.dictionary);
     var ignoreLiteral = config.ignore;
     var queue = 0;
```

Comment:
Everything is sync now, so you can/must remove all the `next` and `queue` related things!